psi4/driver/procrouting/proc.py

```diff
@@ -786,6 +786,30 @@ def select_ccsd_gradient(name, **kwargs):
     else:
         return func(name, **kwargs)
 
+def select_cc2_gradient(name, **kwargs):
```

Comment:
Yup, we don't need this function as of now and I have removed it. I have updated the docs as you suggested. Thanks again!